ydb/docs/en/core/dev/troubleshooting/performance/queries/overloaded-errors.md

```diff
@@ -0,0 +1,18 @@
+# Overloaded errors
+
+{{ ydb-short-name }} returns `OVERLOADED` errors in the following cases:
+
+* Overloaded table partitions with over 10000 operations in their queue.
```

Comment:
there is no such limit. Where did you find this?

Reply:
Updated information about the limits